sidebars.js

```diff
@@ -17,42 +17,106 @@ const sidebars = {
   //documentationSidebar: [{ type: 'autogenerated', dirName: '.' }],
 
   // But you can create a sidebar manually
+  startSidebar : [
+    {
+      type: 'category',
+      label: 'Getting Started',
+      collapsible: false,
+      collapsed: false,
+      items: [
+        'get-started/index',
+        {
+          type: 'link',
+          label: 'About Lisk', // The link label
+          href: '/category/about-lisk', // The internal path
+        },
+        {
+          type: 'link',
+          label: 'Lisk Tools', // The link label
+          href: '/category/lisk-tools', // The internal path
+        },
+      ],
+      link: {
+        type: 'generated-index',
+        keywords: ['getting started','start', 'overview'],
+      },
+    },
+  ],
 
   documentationSidebar: [
+   /*  {
+      type: 'doc',
+      id: 'overview', // document ID
+      label: 'Getting started', // sidebar label
+    }, */
     {
       type: 'category',
       label: 'About Lisk',
-      collapsible: true,
+      collapsible: false,
       collapsed: false,
       items: [
-        'intro',
-        { type: 'ref', id: 'docs-user/connecting-to-a-wallet' },
-        'about-lisk/network-info',
-        'about-lisk/fees',
-        'about-lisk/contracts'
+        'lisk-chain/about-lisk/network-info',
+        'lisk-chain/about-lisk/intro',
+        'lisk-chain/about-lisk/fees',
+        'lisk-chain/about-lisk/contracts'
       ],
+      link: {
+        type: 'generated-index',
+        description: 'General information about the Lisk chain',
+        keywords: ['Lisk','About Lisk', 'Lisk info', 'fees', 'network info'],
+      },
     },
     {
       type: 'category',
-      label: 'Building on Lisk',
+      label: 'Lisk Tools',
       collapsible: true,
       collapsed: false,
       items: [
-        'building-on-lisk/web3-app-development',
+        'lisk-chain/lisk-tools/deployed-tokens',
+        'lisk-chain/lisk-tools/exchanges',
+        'lisk-chain/lisk-tools/wallets',
+        'lisk-chain/lisk-tools/bridges',
+        'lisk-chain/lisk-tools/node-providers',
+        'lisk-chain/lisk-tools/faucets',
+        'lisk-chain/lisk-tools/interoperability',
+        'lisk-chain/lisk-tools/oracles',
+        'lisk-chain/lisk-tools/utilities',
+        'lisk-chain/lisk-tools/indexers'
+      ],
+      link: {
+        type: 'generated-index',
+        keywords: ['tools','resources'],
+      },
+    }
+  ],
+
+  pannaSidebar: [
+    'panna/panna-intro'
+  ],
+
+  guidesSidebar: [
+    {
+      type: 'category',
+      label: 'Developer Guides',
+      collapsible: false,
+      collapsed: false,
+      items: [
+        'guides/web3-app-development',
+        'guides/connecting-to-a-wallet',
+        'guides/import-lsk',
```

Comment:
These two guides are relevent for non-developers as well and need to be in a more prominent place. I would move them to the "Get Started" Section so it would look like this:

About Lisk
Lisk Tools
Connecting Lisk to a Wallet
Importing LSK to MetaMask